src/autocomplete/__tests__/getAutocompleteFilter-test.js

```diff
@@ -11,6 +11,12 @@ describe('getAutocompleteFilter', () => {
     selection = { start: 1, end: 1 };
     expect(getAutocompleteFilter('@', selection)).toEqual({ filter: '', lastWordPrefix: '@' });
 
+    selection = { start: 1, end: 1 };
```

Comment:
These would be better in a new test case.

Reply:
Done 👍